python/py-sentence-transformers/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        UKPLab sentence-transformers 0.2.4 v
+revision            0
+name                py-${github.project}
+categories-append   textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Sentence Embeddings with BERT & XLNet
+
+long_description    BERT/XLNet produces out-of-the-box rather bad\
+                    sentence embeddings. This repository fine-tunes\
+                    BERT/XLNet with a siamese or triplet network\
+                    structure to produce semantically meaningful\
+                    sentence embeddings that can be used in\
+                    unsupervised scenarios: Semantic textual\
+                    similarity via cosine-similarity, clustering,\
+                    semantic search. We provide an increasing number\
+                    of state-of-the-art pretrained models that can be\
+                    used to derive sentence embeddings. See Pretrained\
+                    Models. Details of the implemented approaches can\
+                    be found in our publication: Sentence-BERT:\
+                    Sentence Embeddings using Siamese BERT-Networks\
+                    (published at EMNLP 2019). You can use this code\
+                    to easily train your own sentence embeddings, that\
+                    are tuned for your specific task. We provide\
+                    various dataset readers and you can tune sentence\
+                    embeddings with different loss function, depending\
+                    on the structure of your dataset. For further\
+                    details, see Train your own Sentence Embeddings.
+
+checksums           rmd160  00d5fb33f40ae1ad4f3fb1fda8f6cf3a4c28f92f \
+                    sha256  4bbeca502b8b62f9011518135d958129735ab34b1c5576e922d84cb5abf469d4 \
+                    size    48748
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-nltk \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-pytorch \
+                    port:py${python.version}-scikit-learn \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-transformers
```

Comment:
this seems to be missing ```py-tqdm```see [here](https://github.com/UKPLab/sentence-transformers/blob/5cacadb058234af54d701f072c17559abb6918ac/setup.py#L20)

Reply:
Thanks done.